coordinator/coordinator.go

```diff
@@ -228,6 +228,19 @@ func NewCoordinator(meta *metadata.Client, logger *zap.SugaredLogger, cli *clien
 
 const MAX_ATTEMPTS = 3
 
+func (c *Coordinator) checkError(err error, jobName string) {
+	switch err.(type) {
+	case JobDoesNotExistError:
+		c.Logger.Info(err)
+	case ResourceAlreadyFailedError:
+		c.Logger.Infow("resource has failed previously. Ignoring....", "key", jobName)
```

Comment:
Does the ignoring correlate to any functionality implied to be made or is it just a convention for the reader to ignore the error?

Reply:
Doesnt change functionality. It removes the "Job failed: job already failed" error which is confusing and changes it to Info instead of erroring again